MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -41,6 +50,8 @@ protected BasePeriodicWebSocketListener(ILogger<BasePeriodicWebSocketListener<TR
             ArgumentNullException.ThrowIfNull(logger);
 
             Logger = logger;
+
+            _ = HandleMessages();
```

Comment:
Discarding the task without explicit error handling in `HandleMessages` is an antipattern and should be avoided

Reply:
There's a try catch?

Reply:
If `_channel.Reader.ReadAllAsync()` throws an exception it gets swallowed